tests/test_html.py

```diff
@@ -397,6 +397,307 @@ def test_non_entity_unicode_character(self):
         self.assert_document_generates_html(document, expected_html)
 
 
+class FakedSubScriptTestCase(DocumentGeneratorTestCase):
+    style_xml = '''
+        <style styleId="foo" type="paragraph">
```

Comment:
I agree. However, you can create custom styles, and `foo` is an acceptable name for such a style. The name you use is also used as the `styleId`.